### PR TITLE
Correct example text regarding GeoIP

### PR DESCRIPTION
**Pull request by fzipi**
- Add maxmind tool for downloading files.
- Add text to point to the proper file depending on ModSecurity version.

### CODE REVIEW

**Comment by fzipi** [comment]
This one should fix #1303 .

**Comment by fzipi** [comment]
Seems like `.dat` support was ended this year. Will recheck and add more documentation or remove it.

**Comment by dune73** [comment]
Yes, it's a pity. @emphazer transforms the new format into the old format. I migrated my stuff to mod_maxmind and then I use the geo code in the env variable in ModSec.

I doubt we want to publish the conversion script with CRS, but it would probably be a good idea to explain the situation and the options in the docs.

**Comment by fzipi** [comment]
@fgsch Updated text with one specific tool for the update. We can write also a new blog post showing the usage of this tool (or even emphazer's script), and afterwards link the config file example to it.

File: crs-setup.conf.example [hunk 1]
```diff
@@ -622,20 +622,49 @@ SecDefaultAction "phase:2,log,auditlog,pass"
...
 # To use geolocation, we make use of the MaxMind GeoIP database.
 # This database is not included with the CRS and must be downloaded.
-# You should also update the database regularly, for instance every month.
-# The CRS contains a tool to download it to util/geo-location/GeoIP.dat:
-#   util/upgrade.py --geoip
 #
-# This product includes GeoLite data created by MaxMind, available from:
-# http://www.maxmind.com.
+# There are two formats for the GeoIP database. ModSecurity v2 uses v1 (.dat files),
+# and ModSecurity v3 uses v2 (.mmdb files).
```
Comment (fgsch) [inline crs-setup.conf.example]: Instead of v1 and v2 can we say GeoLite and GeoLite2 here?

Comment (fzipi) [reply crs-setup.conf.example]: Yes, it is better that way. Otherwise it is confusing.

**Comment by fgsch** [comment]
LGTM 👍

**Pull request merged**